src/components/DateInput/NavArrows.tsx

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import Flex from 'components/Flex';
 import FlexItem from 'components/FlexItem';
 import IconButton from 'components/IconButton';
 import ActionsArrowLeftIcon from 'icons/react/ActionsArrowLeftIcon';
 import ActionsArrowRightIcon from 'icons/react/ActionsArrowRightIcon';
 
+export interface NavArrowProps {
+  /** callback for clicking on left arrow */
+  onPreviousClick?: () => void;
+
+  /** callback for clicking on right arrow */
+  onNextClick?: () => void;
+}
 /**
  * Private component for DateInput that renders prev/next arrows.
  * @param {Object} props react props
  * @returns {ReactElement}
  */
-const NavArrows = ({ onPreviousClick, onNextClick }) => (
+const NavArrows = ({ onPreviousClick, onNextClick }: NavArrowProps): JSX.Element => (
   <div className="fdsDateInput-navArrows alignChild--right--center">
     <Flex noGutters>
       <FlexItem shrink>
         <IconButton
-          onClick={() => onPreviousClick()}
+          onClick={() => onPreviousClick?.()}
```

Comment:
I've never seen this pattern before 🙃 would it be cleaner to just assign a `noop` as a default parameter to these 2 parameters? That way we don't have to optional chain here?

Reply:
that's an alternative, yes, and probably is more readable, but in the end i guess it's a matter of preference, i'm ok with either, just optional chaining seemed like an acceptable solution to me, but we can change these to `noop`, but if we're doing that, i'd suggest we declare a `noop` at `src\util` maybe, so we just pull it from there every place we need it.

Reply:
`src/util/noop` strategy sounds good!